.travis.yml

```diff
@@ -63,6 +63,9 @@ script:
   - if [[ "$WERROR" != "true" ]]; then
       python build_total.py ecl || travis_terminate 1;
     else
+      git clone https://github.com/Statoil/libecl.git || travis_terminate 1;
```

Comment:
This PR was not yet for anyone else to look at ;) The existing changes was just a quick try at getting the warnings to be errors at Travis. The plan is to have the build_total.py take arguments that allows one to set the appropriate flags..